scratch/copy-mk.py

```diff
@@ -111,6 +111,10 @@ async def async_main(args) -> int:
     sync_time = await corr2_client.sensor_value("sync-time", float)
     channels = await corr2_client.sensor_value("antenna-channelised-voltage-n-chans", int)
     sample_rate = await corr2_client.sensor_value("adc-sample-rate", float)
+    # This returns a string representation of a list of tuples, with each item
+    # in the format (input-label, input-index, LRU host, index-on-host).
+    input_labelling_str = await corr2_client.sensor_value("input-labelling", str)
+    input_labels_list = [labels_tuple.split(",")[0][1:-1] for labels_tuple in input_labelling_str[1:-1].split("(")[1:]]
```

Comment:
Rather than trying to parse the string yourself, you can let Python do it. Import `ast`, then something like
```
input_labelling = ast.literal_eval(input_labelling_str)
input_labels_list = [label[0] for label in input_labelling]
```

Reply:
I'd also suggest just calling this `input_labels`. We don't generally bundle the type of a variable into the name unless it's needed for disambiguation or it's unexpected - for a variable with a plural name it's not unexpected that it's a list.

Reply:
Apologies, you are quite right. I had `input_labels_str` underneath it before I moved the `",".join(...)` to be in-line in the config dict.